CONTRIBUTING.md

```diff
@@ -6,8 +6,9 @@ See [When Things Go Wrong With FontForge Itself](http://designwithfontforge.com/
 
 ## Contributing Code
 
-We request that all pull requests be actively reviewed by at least one other developer, and in 2014 and for 2015, Frank Trampe has volunteered to do that if no one else gets there first. 
-He aims to review your PR within 1 week of your most recent commit.
+We request that all pull requests be actively reviewed by at least one other developer.
+Developer volunteers monitor the pull request queue, but the time it takes for a given
+review depends on what files are edited and the degree of change. 
```

Comment:
Our effective policy is stricter than stated here, and we ought to alter the wording to reflect that. Pull requests must be reviewed by an active core contributor (@skef, @jtanx, @JoesCat, @khaledhosny, or me), with the understanding that each of us knows if something is outside his sphere of knowledge. We lost a lot of reliability and a lot of reputation in the days of gratuitous merges, and we need to continue to make sure that anybody reviewing a pull request actually understands what it does and thinks that the changes are strictly positive.

Maybe something like this.

> Merging into the master branch requires review and approval of the pull request by an active core contributor to the project. The reviewer must fully comprehend the code prior to merging it, and any changes must be strictly positive unless discussed in advance on the mailing list. FontForge is an extremely complex piece of software, and keeping it in top form requires this level of care. Please understand that large pull requests may take a long time to be approved or may be declined, even if well written, due to limited code review resources.

Reply:
I put in something close to this, but as a novice I might interpret "strictly positive" as "only adding lines, not removing them", which (I assume) is not what we're trying to get across. Do you mean something like "any changes that remove functionality must be discussed in advance on the mailing list"? Something more general?

Reply:
Maybe say "non-regressive and non-breaking"?

Reply:
Done